workflow/rules/common.smk

```diff
@@ -459,6 +462,82 @@ def get_adapters(wildcards):
     return config["adapters"]["illumina-revelo"]
 
 
+def get_final_assemblies(wildcards):
+    if wildcards.assembly_type == "masked-assembly":
+        pattern = "results/{{date}}/contigs/masked/{sample}.fasta"
+    elif wildcards.assembly_type == "pseudo-assembly":
+        pattern = "results/{{date}}/contigs/pseudoassembled/{sample}.fasta"
+
+    return expand(pattern, sample=get_samples_for_date(wildcards.date))
+
+
+def get_final_assemblies_identity(wildcards):
+    if wildcards.assembly_type == "masked-assembly":
+        pattern = "results/{{date}}/quast/masked/{sample}/report.tsv"
+    elif wildcards.assembly_type == "pseudo-assembly":
+        pattern = "results/{{date}}/quast/pseudoassembly/{sample}/report.tsv"
+
+    return expand(pattern, sample=get_samples_for_date(wildcards.date))
+
+
+def get_assemblies_for_submission(wildcards, agg_typ):
+    if wildcards.date != BENCHMARK_DATE_WILDCARD:
+        with checkpoints.rki_filter.get(
+            date=wildcards.date, assembly_type="masked-assembly"
+        ).output[0].open() as f:
+            masked_samples = pd.read_csv(f, squeeze=True, header=None).astype(str).to_list()
+
+        with checkpoints.rki_filter.get(
+            date=wildcards.date, assembly_type="pseudo-assembly"
+        ).output[0].open() as f:
+            pseudo_samples = pd.read_csv(f, squeeze=True, header=None).astype(str).to_list()
+    # for testing of pangolin don't create pseudo-assembly
+    else:
+        masked_samples = [wildcards.sample]
+
+    pseudo_assembly_pattern = "results/{{date}}/contigs/pseudoassembled/{sample}.fasta"
+    normal_assembly_pattern = "results/{{date}}/contigs/masked/{sample}.fasta"
+
+    # get accepted sampels for rki submission
+    if agg_typ == "accepted samples":
```

Comment:
Variable name should be english: agg_type

Reply:
Thanks for spotting this! Addressed in a15f831.